deploy/scripts/src/cri-lime/test.ts

```diff
@@ -219,53 +221,61 @@ export function fraud (): void {
   const encodedCredentials = encoding.b64encode(credentials)
   iterationsStarted.add(1)
 
-  res = group(groups[0], () => timeRequest(() => // B01_Fraud_01_CoreStubEditUserContinue
-    http.post(
-      env.ipvCoreStub + '/edit-user',
-      {
-        cri: `fraud-cri-${env.envName}`,
-        rowNumber: '197',
-        firstName: userDetails.firstName,
-        surname: userDetails.lastName,
-        'dateOfBirth-day': `${userDetails.day}`,
-        'dateOfBirth-month': `${userDetails.month}`,
-        'dateOfBirth-year': `${userDetails.year}`,
-        buildingNumber: `${userDetails.buildNum}`,
-        buildingName: userDetails.buildName,
-        street: userDetails.street,
-        townCity: userDetails.city,
-        postCode: userDetails.postCode,
-        validFromDay: '26',
-        validFromMonth: '02',
-        validFromYear: '2021',
-        validUntilDay: '',
-        validUntilMonth: '',
-        validUntilYear: '',
-        'SecondaryUKAddress.buildingNumber': '',
-        'SecondaryUKAddress.buildingName': '',
-        'SecondaryUKAddress.street': '',
-        'SecondaryUKAddress.townCity': '',
-        'SecondaryUKAddress.postCode': '',
-        'SecondaryUKAddress.validFromDay': '',
-        'SecondaryUKAddress.validFromMonth': '',
-        'SecondaryUKAddress.validFromYear': '',
-        'SecondaryUKAddress.validUntilDay': '',
-        'SecondaryUKAddress.validUntilMonth': '',
-        'SecondaryUKAddress.validUntilYear': ''
-      },
-      {
-        headers: { Authorization: `Basic ${encodedCredentials}` }
-      }
-    ),
-  { isStatusCode200, ...pageContentCheck('We need to check your details') }))
+  group(groups[0], () => { // B01_Fraud_01_CoreStubEditUserContinue
+    timeRequest(() => {
+      res = group(groups[1].split('::')[0], () => timeRequest(() => // 01_CoreStubCall
```

Comment:
```suggestion
      res = group(groups[1].split('::')[1], () => timeRequest(() => // 01_CoreStubCall
```

Reply:
The `split` function will split the below in 2 parts (based on `::`)

`B01_Fraud_01_CoreStubEditUserContinue::01_CoreStubCall`

The string at the index 0 will be `B01_Fraud_01_CoreStubEditUserContinue` and string at index 1 will be `01_CoreStubCall` so we need to use the correct index.